experimental/veggieseasons/lib/styles.dart

```diff
@@ -216,12 +216,20 @@ abstract class Styles {
 
   static const Color iconGold = Color(0xffdba800);
 
+  static const Color iconRed = Color(0xffFF0000);
```

Comment:
nit: Use lowercase letters here, since that's what the other colors are doing.

Actually, one of the colors on line 209 *isn't* doing that, so if you have a second, change that one as well :)

Reply:
Okay, changed it and also at line 249 :)